crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1711,6 +1711,15 @@ impl Client {
         Ok(joined_room)
     }
 
+    pub(crate) async fn create_dm_room(&self, user_id: &UserId) -> Result<room::Joined> {
```

Comment:
Perhaps we should make this public, this way the CI won't complain about dead code. Maybe rename it into `create_dm()`? And of course document it.